### DISCUSSION TITLE
Llegan los commits que hago?

### DISCUSSION THREAD

**Discussion by jjbagus** [atralice/Curso.Prep.Henry#49] [Q&A]
Hola chicos, mi nombre es Francisco, simplemente quería saber si los commits que hago llegan a donde tienen que hacerlo, o si me está faltando hacer algún paso más. Gracias!

**Comment by atralice** [comment]
Hola, Fran.
Te tenes que fijar en tu github, si ves los cambios que hiciste de manera local en github entonces está ok.

**Reply by cristianrafael16** [reply]
Hola Tony! como va? Che te hago una pregunta. Perdón que no sea el hilo. Pero no se como encontrarlos en slac. Me darías una mano para poder hablarles por ahí? gracias!
